src/components/TimelineOnboarding.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { computed, watch } from 'vue';
+import { useI18n } from '@/composables/useI18n';
+import { useOnboardingChecklist } from '@/composables/useOnboardingChecklist';
+import { useStorage } from '@vueuse/core';
+
+const props = defineProps<{
+  web3Account: string;
+}>();
+
+const { t } = useI18n();
+const { onboardingChecklist } = useOnboardingChecklist();
+
+const hideOnboarding = useStorage(
+  `snapshot.hideOnboarding.${props.web3Account.slice(0, 8).toLowerCase()}`,
+  false
+);
+
+const checkListCompleted = computed(() => {
+  let completed = true;
+
+  onboardingChecklist.value.forEach(checkListItem => {
+    if (!checkListItem.checked) {
+      completed = false;
+      return;
+    }
+  });
+
+  return completed;
+});
+
+watch(checkListCompleted, completed => {
```

Comment:
Instead of auto hiding it on completion we could have a share on twitter button

Reply:
I think it could be a good idea. I think it should be also discussed with @bonustrack. 

It will ask some UX thinking about how to handle the closing of this panel.